### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by clubanderson**
## Related issue(s)

Fixes #

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# edge-mc
+Edge Multi-Cluster
+
```
Comment (pdettori) [inline README.md:3]: I think we should start by saying that this is a subproject of kcp that aims to address the concerns of Edge Multi Cluster scenarios

Comment (pdettori) [reply README.md:3]: Also, I suggest we follow an outline similar to the main kcp project - something along the lines of 

- Overview
- Quickstart
- Next steps
- Contributing

We may not have yet code to try in Quickstart, but I think what might be of interest for developers browsing this site will be something to try out.

**Comment by ezrasilvera** [comment]
Can we just use Edge instead of Edge-mc ?

**Comment by clubanderson** [comment]
discarding -  superseded by #13

**Pull request closed without merging**